schemas/content-fragments/hotels/holidays.ts

```diff
@@ -0,0 +1,30 @@
+export default {
+    title:"Holidays",
+      name: 'holidays',
+      type: "document",
+      fields: [
+        {
+          name: "title",
+          title: "Title",
+          type: "string",
+        },
+        {
+            name: "sectionTitle",
+            title: "Section Title",
+            type: 'title',
+        },
+        {
+          name: "description",
+          title: "Description",
+          type: "string",
+        },
+        {
+          name: "holidayDetails",
```

Comment:
Both type and name are the same. Could you provide a different vocabulary for the name?

Reply:
resolved , please check